src/cart/auth/auth.service.ts

```diff
@@ -0,0 +1,13 @@
+import { loginUser, registerUser } from './auth.repository';
+
+export class AuthService {
+    async login(creds: { email: string, password: string }) {
+        return await loginUser(creds);
+    }
+
+    async register(userDTO: object) {
```

Comment:
I would reccomend to create TypeScript type for the user instead of just assigning `object` type. For example:

```
interface User {
 ...
}
```